prime-router/examples/cdc-elims-metabase-queries/active-stlts.sql

```diff
@@ -0,0 +1,9 @@
+select  org.name || '.' || receiver.name as "Receiver Name", receiver.created_at as "Created At"
+from setting as receiver
+join setting org on org.setting_id = receiver.organization_id
+where receiver."type" = 'RECEIVER'
+and receiver."values" ->> 'topic' = 'elr-elims'
+and receiver.is_active = true
+and receiver."values" ? 'transport'
+and receiver.is_deleted = falsea
```

Comment:
```suggestion
and receiver.is_deleted = false
```

Reply:
Thanks, fat finger from copying out of metabase!